docs/howto.md

```diff
@@ -89,15 +82,19 @@ Make sure you have completed the steps for installation and Google Cloud setup d
 
 Replace `[YOUR_BUCKET_NAME]` with the name of the bucket you created.
 
-5. Run the pipeline:
+5. Run the pipeline using caper:
 
-```
-  $ java -jar -Dconfig.file=backends/backend.conf -Dbackend.default=google -Dbackend.providers.google.config.project=[YOUR_PROJECT] -Dbackend.providers.google.config.root=gs://[YOUR_BUCKET_NAME]/output cromwell-34.jar run rna-seq-pipeline.wdl -i input.json -o workflow_opts/docker.json
+```bash
+  $ caper run rna-seq-pipeline.wdl -i input.json -b gcp -m testrun_metadata.json
 ```
 
-Replace `[YOUR_PROJECT]` with the project id of the project you created and `[YOUR_BUCKET_NAME]` with the name of the bucket you created.
+6. Run croo, to easier view the pipeline output:
```

Comment:
Run croo, to make finding outputs easier.

Reply:
+1